lambda/src/main/scala/pricemigrationengine/model/AmendmentData.scala

```diff
@@ -293,7 +293,7 @@ object PriceCapper {
     List(estimatedNewPrice, oldPrice * priceCappingMultiplier).min
 
   def priceCorrectionFactor(oldPrice: BigDecimal, estimatedNewPrice: BigDecimal): BigDecimal = {
-    if (estimatedNewPrice == 0 || estimatedNewPrice < oldPrice * priceCappingMultiplier) {
+    if (estimatedNewPrice == 0 || estimatedNewPrice.compareTo(oldPrice * priceCappingMultiplier) <= 0) {
```

Comment:
```suggestion
    if (estimatedNewPrice == 0 || estimatedNewPrice.compareTo(oldPrice.multiply(BigDecimal(priceCappingMultiplier))) <= 0) {
```

Reply:
<img width="1414" alt="Screenshot 2023-03-16 at 11 30 43" src="https://user-images.githubusercontent.com/6035518/225604275-a2a7eec6-2b18-4231-ad74-6b28f7a142fd.png">

Reply:
@kelvin-chappell

Reply:
I was looking at `java.math.BigDecimal` rather than `scala.math.BigDecimal`.
The former has a [multiply](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/math/BigDecimal.html#multiply%28java.math.BigDecimal%29) method but the latter uses [*](https://www.scala-lang.org/api/2.12.6/scala/math/BigDecimal.html#*(that:scala.math.BigDecimal):scala.math.BigDecimal) for the same thing.
So I withdraw this suggestion, as it's redundant.